lib/views/message.coffee

```diff
@@ -43,11 +40,22 @@ class Message extends HTMLElement
         else
           el.appendChild message.html
     else if message.multiline
-      el.appendChild MultilineElement.fromText message.text
+      el.appendChild Message.processMultiLine message.text
     else
       el.textContent = message.text
     el
 
+  @processMultiLine: (text)->
+    container = document.createElement 'div'
+    container.classList.add 'linter-multiline-message'
```

Comment:
Because in most cases, classes are redundant. I strive for sensible minimalism in everything (not that I'm particularly good at it, but I try)